src/raster/r.damflood/main.c

```diff
@@ -281,10 +281,10 @@ int main(int argc, char *argv[]){
   opt.met->answer = "dambreak-without_hypotesis";
   //opt.met->guisection  = _("Options");
 
-  /* LEGENDA
-  total_dambreak-without_hypotesis = nessuna ipotesi sulla velocita' iniziale
-  total_dambreak = Hp altezza critica --> velocita' iniziale (h critica) = 0.93*sqrt(h);
-  small_dam_breach = Hp stramazzo --> velocita' iniziale = 0.4*sqrt(2*g*h)
+  /* LEGEND
+  total_dambreak-without_hypotesis = No hypothesis about initial velocity
+  total_dambreak = Hp critical height --> initial velocity (critical h) = 0.93*sqrt(h);
+  small_dam_breach = Hp dam --> initial velocity = 0.4*sqrt(2*g*h)
```

Comment:
The descriptions in this legend don't even match with the options displayed a couple lines above (generated in the manual). What is the value of these comments?